3rdparty/python/requirements.txt

```diff
@@ -9,14 +9,15 @@ click
 confluent-kafka
 cookiecutter==1.7.3
 hypothesis
+lark-parser
 opentelemetry-api
 opentelemetry-exporter-zipkin-json
 opentelemetry-instrumentation
 opentelemetry-instrumentation-botocore
 opentelemetry-sdk
 protobuf>=3.15.8,<3.16.0
-pulumi>=3.0.0,<4.0.0
-pulumi-aws>=5.0.0,<6.0.0
+pulumi==3.33.1
```

Comment:
I'm locking these since the upgrade causes pulumi preview/up to break. One bug at a time